dependencies.props

```diff
@@ -10,40 +10,37 @@
   -->
   <PropertyGroup>
     <WCFCurrentRef>bc189e7681f4e64bb2ba779eefb40b9e4542c683</WCFCurrentRef>
+    <CoreFxCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreFxCurrentRef>
+    <CoreClrCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreClrCurrentRef>
+    <CoreSetupCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreSetupCurrentRef>
     <StandardCurrentRef>c520a2569b40fc53cf51e4f6970c3e7411adc173</StandardCurrentRef>
     <BuildToolsCurrentRef>8fc22d261883eb7a1f8765bfe716d806303d0318</BuildToolsCurrentRef>
   </PropertyGroup>
 
-  <!-- Auto-upgraded properties for each build info dependency. -->
+  <!-- Product dependency versions. -->
   <PropertyGroup>
-    <WCFExpectedPrerelease>preview3-26429-01</WCFExpectedPrerelease>
-    <CoreFxExpectedPrerelease>preview3-26430-04</CoreFxExpectedPrerelease>
     <NETStandardPackageVersion>2.0.3</NETStandardPackageVersion>
     <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
-    <!--
-      This property is used for many CoreFX packages, because they have the same
-      versions. Make more properties for CoreFX if split versions are required.
-    -->
+    <CoreFxProductDependencyVersion>4.5.0</CoreFxProductDependencyVersion>
+  </PropertyGroup>
+
+  <!-- Tests/infrastructure dependency versions. -->
+  <PropertyGroup>
+    <WCFExpectedPrerelease>preview3-26429-01</WCFExpectedPrerelease>
```

Comment:
Can this be removed (in favor of the full-version props)? Prerelease spec properties aren't ideal, since usually they're combined with some static `x.x.x` and this causes restore problems when moving to a new major/minor/patch.

Reply:
Sure, I'll take a look at that.